Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -103,6 +104,14 @@ export default class LinearTimelineComponent extends Vue {
             self.setPageFromDate(eventDate);
         });
 
+        this.eventBus.$on(
+            EventMessageName.TimelineCurrentDateUpdated,
+            function (currentDate: Date) {
+                self.setPageFromDate(currentDate);
+                // self.currentPage = 3; // debug only, to be updated when Leo works on 8693
```

Comment:
is this the ticket?

Reply:
8693 is the next one that I will work on. This PR is pretty much to notify the linearTimeLine from Calendar vue

Reply:
I think this ticket about implementing the switch between month view to timeline view. 8693 is about timeline view to calendar view.

Reply:
right, I though the 8693 is supposed to to both. I will implement code for this